subsys/bluetooth/audio/pacs.c

```diff
@@ -201,17 +201,17 @@ static ssize_t supported_context_read(struct bt_conn *conn,
 {
 	struct bt_pacs_context context;
 
-	if (IS_ENABLED(CONFIG_BT_PAC_SNK)) {
-		context.snk = sys_cpu_to_le16(CONFIG_BT_PACS_SNK_CONTEXT);
-	} else {
-		context.snk = BT_AUDIO_CONTEXT_TYPE_PROHIBITED;
-	}
+#if defined(CONFIG_BT_PAC_SNK)
```

Comment:
Just wondering what prompted you to change from the IS_ENABLED to #if defined here

Reply:
The Kconfig `CONFIG_BT_PACS_SNK_CONTEXT` is guarded by `CONFIG_BT_PAC_SNK`:
```c
if BT_PAC_SRC
config BT_PACS_SRC_CONTEXT
	hex "Supported Source Contexts"
	default 0x03ff
	range 0x0000 0x03ff
```

So we won't be able to use it unless `CONFIG_BT_PAC_SNK` is enabled (`IS_ENABLED` does not work for guarded Kconfigs)